### PR TITLE
Configure the app with django

### PR DESCRIPTION
**Pull request by icemoon4**
Created a fresh Django app (copied in some of the old files in case we still need those). Configured the DATABASE settings to connect to Robotic Dogs. Used `inspectdb` to copy the database tables into a models.py file. Reorganized the file and made adjustments required for the Django server to start and be able to query tables. Used `black` and `isort` to format the python files.

### CODE REVIEW

File: dsa_actionkit/aktemplates.py [hunk 1]
```diff
@@ -2,12 +2,14 @@
 import os
 import sys
 
-#STATIC_FALLBACK="/static/js/fallback_local" PYTHONPATH=./djanger django-admin runserver --settings=settings
+# STATIC_FALLBACK="/static/js/fallback_local" PYTHONPATH=./djanger django-admin runserver --settings=settings
 
 
 def serve_templates():
     os.environ.setdefault("DJANGO_SETTINGS_MODULE", "dsa_actionkit.settings")
```
Comment (swoopej) [inline dsa_actionkit/aktemplates.py:9]: is there a reason to nest the settings for the project under the dsa_actionkit directory/app?  That might be best practice, just checking.

Comment (swoopej) [reply dsa_actionkit/aktemplates.py:9]: (maybe this is standard practice in django these days, if so then ignore me!)

**Comment by swoopej** [comment]
settings.py changes look good to me

File: dsa_actionkit/aktemplates.py (hunk 1 above)
Comment (icemoon4) [reply dsa_actionkit/aktemplates.py:9]: This is one of the original files from the directory, I just moved it over. Honestly didn't even really look at it haha, I just didn't want to delete anything that we might still need. Updates to this file are just a result of auto formatting that I did

File: dsa_actionkit/dsa_actionkit/settings.py [hunk 2]
```diff
@@ -0,0 +1,150 @@
...
+# https://docs.djangoproject.com/en/3.2/ref/settings/#databases
+
+DATABASES = {
+    "default": {
+        "ENGINE": "django.db.backends.mysql",
+        "NAME": "ak_roboticdogs",
+        "USER": "USERNAME",
+        "PASSWORD": "ROBOTIC_DOGS_SECRET_KEY",
+        "HOST": "roboticdogs.client-db.actionkit.com",
+    }
```
Comment (erikdstock) [inline dsa_actionkit/dsa_actionkit/settings.py:69]: can we load these from a .env file instead of keeping a placeholder value in git?

**Review by erikdstock — COMMENTED** [review]
One minor comment, but overall looks good to me - or rather I'm not a pythonist in my day job so it's mostly educational to see others' comments.

File: dsa_actionkit/dsa_actionkit/settings.py (hunk 2 above)
Comment (icemoon4) [reply dsa_actionkit/dsa_actionkit/settings.py:69]: yeah i was thinking we could move to that (noted in the readme) if there's some creds we can use that aren't tied to a personal account? (the only creds i have are my own) unless that's the only kind there is.
if it's something each person would need to set up individually then i would need instructions on how to set that up, and then I could add it into the readme as well

Comment (schlich) [reply dsa_actionkit/dsa_actionkit/settings.py:69]: Enabled .env file imports in my follow-up PR.  By default, the project will use the SQLite database. If/when necessary, users can connect to robotic dogs by including their credentials in a .env file and running the [appropriate command](https://docs.djangoproject.com/en/5.0/topics/db/multi-db/)

**Comment by schlich** [comment]
Overall great work!  I made a follow up pull request [here](https://github.com/dsa-ntc/actionkit-template-renderer/pull/23) that addresses some threads here and more carefully merges the new functionality with the old.  Also, more improvements to the README!

**Review by schlich — CHANGES_REQUESTED** [review]
Overall great work! I made a follow up pull request https://github.com/dsa-ntc/actionkit-template-renderer/pull/23 that addresses some threads here and more carefully merges the new functionality with the old. Also, more improvements to the README!

PR made against Ivy's PR for diffing purposes but once this is reviewed by @icemoon4 and or @swoopej we can merge to main.  Let me know if anything looks broken or confusing.

**Review by schlich — APPROVED** [review]

**Comment by schlich** [comment]
@icemoon4 feel free to do the honors if you're good with the PR!

**Pull request merged**